driver-core/src/test/java/com/datastax/driver/core/ClusterInitTest.java

```diff
@@ -116,7 +117,9 @@ public void should_handle_failing_or_missing_contact_points() throws UnknownHost
             // - 0 or 1 for the missing host. We can't know for sure because contact points are randomized. If it's tried
             //   before the live host there will be a connection attempt, otherwise it will be removed directly because
             //   it's not in the live host's system.peers.
+            //noinspection ResultOfMethodCallIgnored
             verify(socketOptions, atLeast(6)).getKeepAlive();
+            //noinspection ResultOfMethodCallIgnored
             verify(socketOptions, atMost(7)).getKeepAlive();
```

Comment:
These are IntelliJ IDEA warnings, I'd rather avoid IDE-specific stuff in the source code as much as possible (I use IDEA as well, but if someone added random comments for another IDE I would find it annoying).